builder/assignmentEnvImageBuilder.go

```diff
@@ -119,7 +134,10 @@ func (assgnEnv *assignmentEnvironment) verifyLanguage() error {
 	return nil
 }
 
-func (assgnEnv *assignmentEnvironment) writeFromBaseImage() {
+// writeFromBaseImage writes the docker instructions to bytes buffer
+// in 'assignmentEnvironmentImageBuilder' instance staring from the base code runner image.
```

Comment:
you can just say **// writeFromBaseImage writes the docker instructions to a buffer.**

Reply:
Done